apps/post-feeder/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.18 as base
```

Comment:
Naming suggestion for service: `feeder-api` - just to make it clear we split the former `api` directory

Reply:
Will do in a separate PR.